certora/specs/ConsistentState.spec

```
@@ -41,3 +45,112 @@ invariant supplyQueueLengthInRange()
 
 invariant withdrawQueueLengthInRange()
     withdrawQueueLength() <= maxQueueLength();
+
+function hasNoBadPendingTimelock() returns bool {
+    uint192 pendingValue;
+    uint64 validAt;
+    pendingValue, validAt = pendingTimelock();
+
+    return validAt == 0 <=> pendingValue == 0;
```

we could verify only this here
```suggestion
    return validAt == 0 => pendingValue == 0;
```